makelib/common.mk

```diff
@@ -132,7 +132,6 @@ endif
 # Standardize Host Platform variables
 HOST_PLATFORM := $(HOSTOS)_$(HOSTARCH)
 SAFEHOSTPLATFORM := $(HOSTOS)-$(SAFEHOSTARCH)
```

Comment:
@AaronME is there a specific reason both `SAFEHOST_PLATFORM` and `SAFEHOSTPLATFORM` exists? And would you have a preference between them?

Reply:
@muvaf note that one use `-` and one uses `_`

Reply:
Wow I did not notice that, reverted the removal. Thanks folks!